core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalConfiguration.java

```diff
@@ -53,6 +58,28 @@ public static RaftJournalConfiguration defaults(ServiceType serviceType) {
         .setPath(new File(JournalUtils.getJournalLocation().getPath()));
   }
 
+  /**
+   * @param clusterAddresses addresses of all nodes in the Raft cluster
+   * @return true if the cluster addresses contain the local IP, false otherwise
+   */
+  private boolean checkContainLocalIp(List<InetSocketAddress> clusterAddresses) {
```

Comment:
nits:
- How about calling this method `containsLocalIp` -- the convention of this codebase is to have `checkFoo` as a method that returns void but throw exceptions if the input is not meeting expectation.
- seems no member variable is really used in this method, in this case, better making it static